src/EnergyPlus/HeatBalanceSurfaceManager.cc

```diff
@@ -7404,6 +7368,9 @@ void CalcHeatBalanceInsideSurf2(EnergyPlusData &state,
         }
     }
 
+    // Reset radiant HVAC heat gain rate output for next timestep
+    state.dataHeatBalSurf->QdotRadHVACInPerArea = 0.0;
```

Comment:
So, here's another place that's re-setting to zero. Again, should be an explicit loop.